src/lib/blockchains/BitShares.js

```diff
@@ -2164,1304 +2330,1124 @@ export default class BitShares extends BlockchainAPI {
-                opStr += "new_parameters: \n";
-                opStr += "   current_fees" + JSON.stringify(op.new_parameters.current_fees) + "\n";
-                opStr += "   block_interval: " + op.block_interval + "\n";
-                opStr += "   maintenance_interval: " + op.maintenance_interval + "\n";
-                opStr += "   maintenance_skip_slots: " + op.maintenance_skip_slots + "\n";
-                opStr += "   committee_proposal_review_period: " + op.committee_proposal_review_period + "\n";
-                opStr += "   maximum_transaction_size: " + op.maximum_transaction_size + "\n";
-                opStr += "   maximum_block_size: " + op.maximum_block_size + "\n";
-                opStr += "   maximum_time_until_expiration: " + op.maximum_time_until_expiration + "\n";
-                opStr += "   maximum_proposal_lifetime: " + op.maximum_proposal_lifetime + "\n";
-                opStr += "   maximum_asset_whitelist_authorities: " + op.maximum_asset_whitelist_authorities + "\n";
-                opStr += "   maximum_asset_feed_publishers: " + op.maximum_asset_feed_publishers + "\n";
-                opStr += "   maximum_witness_count: " + op.maximum_witness_count + "\n";
-                opStr += "   maximum_committee_count: " + op.maximum_committee_count + "\n";
-                opStr += "   maximum_authority_membership: " + op.maximum_authority_membership + "\n";
-                opStr += "   reserve_percent_of_fee: " + op.reserve_percent_of_fee + "\n";
-                opStr += "   network_percent_of_fee: " + op.network_percent_of_fee + "\n";
-                opStr += "   lifetime_referrer_percent_of_fee: " + op.lifetime_referrer_percent_of_fee + "\n";
-                opStr += "   cashback_vesting_period_seconds: " + op.cashback_vesting_period_seconds + "\n";
-                opStr += "   cashback_vesting_threshold: " + op.cashback_vesting_threshold + "\n";
-                opStr += "   count_non_member_votes: " + op.count_non_member_votes + "\n";
-                opStr += "   allow_non_member_whitelists: " + op.allow_non_member_whitelists + "\n";
-                opStr += "   witness_pay_per_block: " + op.witness_pay_per_block + "\n";
-                opStr += "   worker_budget_per_day: " + op.worker_budget_per_day + "\n";
-                opStr += "   max_predicate_opcode: " + op.max_predicate_opcode + "\n";
-                opStr += "   fee_liquidation_threshold: " + op.fee_liquidation_threshold + "\n";
-                opStr += "   accounts_per_fee_scale: " + op.accounts_per_fee_scale + "\n";
-                opStr += "   account_fee_scale_bitshifts: " + op.account_fee_scale_bitshifts + "\n";
-                opStr += "   max_authority_depth: " + op.max_authority_depth + "\n";
-                opStr += "Extensions: " + JSON.stringify(op.extensions) + "\n";
-                opStr += "Estimated fee: " + JSON.stringify(op.fee);
-                operations.push(opStr);
+                currentOperation['rows'] = [
```

Comment:
this class seems to be expoding. it should contain only the key logic to connect to blockchain and handle certains calls. maybe its a good idea to introduce a BitSharesBeautification class separately that is doing only that.

Reply:
Addressed this request in #257 https://github.com/bitshares/beet/pull/257/commits/6399fc5e79f92af706e6c1f5cc621c7737ef90c3